src/routes/targeted-messaging/targeted-messaging.controller.ts

```diff
@@ -0,0 +1,100 @@
+import { TargetedSafeNotFoundError } from '@/domain/targeted-messaging/errors/targeted-safe-not-found.error';
+import {
+  CreateSubmissionDto,
+  CreateSubmissionDtoSchema,
+} from '@/routes/targeted-messaging/entities/create-submission.dto.entity';
+import { Submission } from '@/routes/targeted-messaging/entities/submission.entity';
+import { TargetedMessagingService } from '@/routes/targeted-messaging/targeted-messaging.service';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
+import { NumericStringSchema } from '@/validation/entities/schemas/numeric-string.schema';
+import { ValidationPipe } from '@/validation/pipes/validation.pipe';
+import {
+  Body,
+  Controller,
+  DefaultValuePipe,
+  Get,
+  HttpStatus,
+  NotFoundException,
+  Param,
+  ParseIntPipe,
+  Post,
+  Res,
+} from '@nestjs/common';
+import { ApiCreatedResponse, ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { Response } from 'express';
+
+@ApiTags('targeted-messaging')
+@Controller({
+  path: 'targeted-messaging',
```

Comment:
Do you think we'd have any other form of "messaging" later? We could adjust this to be "/messaging/targets", or drop the "targeted" part.

Reply:
I don't think so tbh. But it depends on product decisions. As this path was done in alignment with product/frontend, I'd let it as it is for now. We could migrate it later if needed though, wdyt?

Reply:
As it's confirmed, we can leave it as is.